Cql/Cql.Packaging/ResourcePackager.cs

```diff
@@ -1,16 +1,15 @@
-﻿using Hl7.Cql.Elm;
-using Hl7.Cql.Graph;
+﻿using Hl7.Cql.Compiler;
 using Hl7.Cql.Packaging.ResourceWriters;
 using Hl7.Fhir.Model;
 using Microsoft.Extensions.Logging;
-using Library = Hl7.Cql.Elm.Library;
 
 namespace Hl7.Cql.Packaging
 {
     /// <summary>
     /// Encapsulates packaging of resources and outputting them using the specified resource writers
     /// </summary>
-    public class ResourcePackager
+    [Obsolete]
```

Comment:
I thought this would eventually be replaced by the ProgramCliPackager, but now I just injected ResourcePackager in the ProgramCliPackager, and LibraryPackager into ResourcePackager.